requirements.txt

```diff
@@ -1,2 +1,2 @@
-netCDF4 >=1.2.0
+netCDF4 >=1.2.0,<1.2.2
```

Comment:
This line did not work with early conda, but it is fine now!

Ping @kwilcox. You asked me recently about standardizing conda and pip requirements. I guess that conda can take pip syntax now as long as the package name is the same. (Both pip and conda are case insensitive BTW.)